src/templates/docs/font-style/index.html

```diff
@@ -42,17 +42,17 @@ <h4 class="f4 mv0 fw6 dib mr4"><%= name %></h4>
       </article>
       <div class="ph3 ph5-ns pt4 pb5">
         <h2 class="f3 bold">Examples</h2>
-        <code class="f6">
+        <code class="f6 gray">
```

Comment:
Some examples used gray as the code colour, I think this works well so I have applied it to all code usage examples in this PR